_data-prepper/simple-pipelines.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Simple pipelines
+nav_order: 12
+---
+
+# Simple pipelines
+
+This tutorial walks you through creating a simple Data Prepper pipeline and how to configure it.
+
+## Installation
+
+First, install Data Prepper with Docker using the following command:
+
+```
+docker pull opensearchproject/data-prepper:latest
+```
+
+## Simple configuration
+
+To help you get started, we have a simple configuration which you can use to run Data Prepper. It generates random strings and writes them to `stdout`. 
+
+Create a Data Prepper configuration file, `data-prepper-config.yaml` and include the following line:
+
+```
+ssl: true
+```
+
+Create a Data Prepper pipeline file, `pipelines.yaml`, with:
+
+```
+simple-sample-pipeline:
+  workers: 2
+  delay: "5000"
+  source:
+    random:
+  sink:
+    - stdout:
+```
+
+## Run Data Prepper
+
+Below are examples for running Data Prepper from the Docker image. If you built from source, you need to make some modifications to the example commands.
+
+For Data Prepper 2.0 or after, use this command:
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD} /data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
+```
+
+For Data Prepper versions before 2.0, use this command:
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml opensearchproject/data-prepper:latest
+```
+
+You should see log output and after a few seconds some UUIDs in the output. It should look something like the following:
```

Comment:
```suggestion
You should see log output and, after a few seconds, some UUIDs in the output. It should look something like the following:
```

Reply:
Is the second instance of "output" the correct word, or do we mean "response"?

Reply:
@natebower Confirmed that it is "output" and not response".